config/jobs/kubernetes/sig-release/release-branch-jobs/1.14.yaml

```diff
@@ -161,36 +161,41 @@ periodics:
       securityContext:
         privileged: true
 - annotations:
+    testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
```

Comment:
Is this actionable to push to on-call?

Reply:
@cblecker -- Merely mimicking the `master` job. This may be the wrong configuration though. Need to get answers on https://github.com/kubernetes/test-infra/pull/15612#issuecomment-566313404 first and potentially rework this.